<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordBox</class>
 <widget class="QDialog" name="PasswordBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>231</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>357</width>
    <height>231</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>357</width>
    <height>231</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Atención</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="passwordLB">
           <property name="text">
            <string>Contraseña:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="passwordLE">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="passwordRepeatLB">
           <property name="text">
            <string>Repita la nueva contraseña: </string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="repeatedPassWordLB">
             <property name="text">
              <string>Contraseña:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="repeatedPassWordLE">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="cancelCipherBT">
           <property name="text">
            <string>Cancelar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cipherBT">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Cifrar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="infoLB">
         <property name="text">
          <string>Sus copias serán cifradas. Ingrese una contraseña a su archivo, por favor.</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="decipherInfoLB">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>El archivo se encuentra cifrado, ingrese la contraseña para decifrarlo.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="decipherPassWordLB">
           <property name="text">
            <string>Contraseña:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="decipherPassWordLE">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="cancelBT">
           <property name="text">
            <string>Cancelar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="decipherBT">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Decifrar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
